charts/kube-master/templates/configmap.yaml

```diff
@@ -85,3 +85,99 @@ data:
     except requests.exceptions.RequestException as e:
       print e
       sys.exit(0)
+{{- if ne .Values.audit "" }}
+  audit-policy.yaml: |-
+    apiVersion: audit.k8s.io/v1
+    kind: Policy
+    rules:
+    - level: None
+      userGroups:
+      - system:nodes
+    - level: None
+      users:
+      - gardener
```

Comment:
I think we can drop all the users with garden in the name. This only applies to gardener clusters